### PR TITLE
🙊🙈 Run eslint Travis tests against .ts and .tsx files too 🙈🙊

### PR DESCRIPTION
**Pull request by estherkim**
Turns out that without including `--ext *.ts,*.tsx` files, eslint will only check JS files

This fixes
- prettier v2 formatting
- updating `.eslintrc` to `.eslintrc.js` because `.eslintrc` is deprecated
- using `path` to set project path in ts configs
- ALL THE LINT ERRORS 🕺

### CODE REVIEW

**Comment by estherkim** [comment]
@rcebulko I see. The child prettier configs get precedence over the root. I'll just delete that for ya.

**Comment by rcebulko** [comment]
@estherkim Strangely when I tried deleting it, it didn't inherit the root as expected, so I added the prop in https://github.com/ampproject/amp-github-apps/pull/775

File: bundle-size-chart/.eslintrc.js [hunk 1]
```diff
@@ -14,7 +16,7 @@
       "modules": true
     },
     "useJSXTextNode": true,
-    "project": "tsconfig.json"    
+    'project': path.resolve(__dirname, './tsconfig.json')
```
[the remark this answers is not in the document]
Comment (estherkim) [reply bundle-size-chart/.eslintrc.js:19]: This way `./tsconfig.json` for each dir can be found by its eslint config even when you're at the root level.

Why JS - https://eslint.org/docs/user-guide/configuring#configuration-file-formats

**Comment by estherkim** [comment]
@rcebulko hmm that's strange. Worked here, PTAL.

**Pull request merged**